tasks/src/main/scala/dagr/tasks/picard/DeleteBam.scala

```diff
@@ -34,18 +34,20 @@ import scala.collection.mutable.ListBuffer
   * Class to remove a BAM file and also the BAI file if it exists.  Succeeds if the files can be deleted,
   * *or* if no files were present to be deleted.
   */
-class DeleteBam(val bam: Path) extends SimpleInJvmTask {
-  name = "DeleteBam." + PathUtil.basename(bam)
+class DeleteBam(val bam: Path*) extends SimpleInJvmTask {
+  name = "DeleteBam" + (if (bam.length == 1) "." + PathUtil.basename(bam.head) else "s." + bam.length)
```

Comment:
This will create names like `DeleteBams.3` right?  Which is a bit confusing since the log and script files get task ID suffixes as well.  Maybe `Delete3Bams`?